src/core.js

```diff
@@ -79,10 +79,15 @@ export const createParser = config => {
           continue;
         }
         if (raw !== null) {
-          styles = merge(
-            styles,
-            parseResponsiveObject(cache.breakpoints, sx, scale, raw, props)
-          );
+          // We have a style class
+          if (raw.valueOf() !== raw) {
```

Comment:
Will also need a test for
```
if (raw.valueOf() !== raw) {
  styles = merge(styles, sx(raw, scale, props));
}
```
while preserving previous behavior

Reply:
I've added some tests. Hopefully they are adequate, but let me know if I missed anything. The actual `valueOf()` resolution happens in our `styled` package in `system.ts` in my PR.